util/test/perf/perfgraph.js

```diff
@@ -71,6 +71,10 @@ var branchInfo = [
                     "releaseDate": "2015-10-01",
                     "branchDate" : "2015-09-24",
                     "revision" : -1}
```

Comment:
Geez, and I even thought to look for missing commas...

I'm just trying to train people not to ask me for reviews... :P